internal/controllers/topology/cluster/structuredmerge/serversidepathhelper.go

```diff
@@ -78,8 +82,6 @@ func NewServerSidePatchHelper(original, modified client.Object, c client.Client,
 				return nil, errors.Wrap(err, "failed to cleanup legacy managed fields from original object")
 			}
 		}
-
-		filterObject(originalUnstructured, helperOptions)
```

Comment:
Q: Why do we not need this anymore?

Reply:
It is getting filtered in `dryRunSSAPatch` and is only here for comparison to the `dryRunUnstructured`